nilearn/plotting/tests/test_find_cuts.py

```diff
@@ -4,6 +4,10 @@
 from nilearn.plotting.find_cuts import (find_xyz_cut_coords, find_cut_slices,
                                         _transform_cut_coords)
 from nilearn._utils.testing import assert_raises_regex, assert_warns
+from sklearn.datasets import load_iris
```

Comment:
Would you mind moving this line up, before the nilearn imports, to separate nilearn imports from the rest.